reliability.md

```diff
@@ -1,33 +1,45 @@
+When making a backup, you must bear in mind that WordPress is made up of two completely different elements: the files and the data. The file part is stored on your server, and can be accessed by FTP or any other method, and the data is stored in the database. This means that when you make a backup you have to do it from both parts, since one does not work without the other.
 
-It's recommended to keep and test regular backups of your WordPress sites using the system-level backup or snapshot infrastructure of your choice. One thing to be aware of is that the code, assets and database change in conjunction with WordPress but may be backed up separately. For this reason, it is a good idea to keep restore points that include backups of code, assets, and database taken at the same point in time. It’s recommended to create restore points before any critical action, e.g. WordPress core update.
+### Where to make copies
 
-## Monitoring
+There are many methods for making backups. The first is the hosting server itself. Usually the providers themselves offer the possibility of making a copy of all the contents hosted on the machine. Usually these copies are made externally on a specific backup server. In this case, you should know how it works and when it is done, as well as how to restore the copy, if it can be partial, total, and if you have access to the data in a simple way.
+
+A second way is manually, through the operating system, so that you mount a system that makes a copy of all files (for example in a ZIP) and on the other hand to dump the database to obtain a copy of the SQL file with all the data.
+
+Finally, and most often (in addition to what the provider can provide) is to have a backup plugin. There are many plugins and in general we recommend that you try several until you find the one that suits your needs. It is not the same to make a backup of a small site (200 MB) as to make a large one (5 GB). The response times and performance of the server could cause the system to become saturated and not perform the copies, so we recommend that you try the different systems until you find the one that works best.
+
+It's also important that when you back up your data, it's stored in at least two different locations. One is usually the server where the website is located. Sometimes the site gets corrupted but it doesn't affect the machine, so having the files close by and available quickly is usually very helpful. The other case is to have them outside the server, in an external place, and even outside the same network or provider or country. The fact that it is in another place would allow that even if that provider had a serious problem (and for example you had no possibility to access it) you could restore your site from an external place.
```

Comment:
This is also phrased in a user-centric way. I like the recommendation to have backups or snapshots in different geographical locations, though!